packages/drivers/odsp-driver/src/odspDocumentService.ts

```diff
@@ -78,7 +79,7 @@ export class OdspDocumentService implements IDocumentService {
         logger: ITelemetryLogger,
         socketIoClientFactory: () => Promise<typeof SocketIOClientStatic>,
         cache: IOdspCache,
-        hostPolicy: HostStoragePolicy,
+        policies: { hostPolicy: HostStoragePolicy; storagePolicy: IDocumentStorageServicePolicies; },
```

Comment:
Names look confusing to me. The previous policy was also storagePolicy supplied by host. Now we have hostStoragePolicy and storagePolicy too but both are storagePolicy. So I think overall it should be called "storagePolicy" which consists of "hostPolicy" and then maybe "LoaderStoragePolicy or DefaultStoragePolicy" or maybe some other name?